### DISCUSSION TITLE
Conventional Commits prefixes not removed on release

### DISCUSSION THREAD

**Discussion by hamzasalaheddine** [jreleaser/jreleaser#1719] [Q&A]
Hello, 

I am currently using JReleaser version 1.14.0 for the release generation and I have noticed that the conventional commits prefixes was not removed on release

Per example, the following commits
 
feat: Add new feature amenis
test: Add IT for feature amenis
docs: update readme with Jreleaser info
ci: Jreleaser conf - Remove announce section

**Will be rendered as**

Changelog
🚀 Features
04a6bf56 feat: Add new feature amenis
🧪 Tests
491dd6c8 test: Add IT for feature amenis
🛠  Build
22ff3c7d ci: Jreleaser conf - Remove announce section
📝 Documentation
a91a811b docs: update readme with Jreleaser info

I use the same configuration present in discussions [#1718](https://github.com/jreleaser/jreleaser/discussions/1718)

Any help or suggestions would be greatly appreciated.

Thank you for your assistance.

Best regards,
Hamza

**Comment by aalmiray** [comment]
I noticed this problem on an early-access release but can't reproduce it anymore.
